contrib/inventory/foreman.ini

```diff
@@ -116,6 +127,8 @@ group_patterns = ["{app}-{tier}-{color}",
 group_prefix = foreman_
 # Whether to fetch facts from Foreman and store them on the host
 want_facts = True
+# Whether to create Ansible groups for host collections
+want_hostcollections = False
```

Comment:
I've changed the default to True and added a comment that it may have to be disabled to work with a stand-alone Foreman server.